packages/stacks-docs/product/components/pagination.html

```diff
@@ -125,90 +126,14 @@
                         </a>
                     </li>
                     <li>
-                        <a class="s-pagination--item" href="#">
-                            Next
-                            <span class="v-visible-sr">page</span>
-                        </a>
+                        <!-- TODO: Change muted to tonal when button PR goes in-->
+                            <a class="s-btn s-btn__muted s-btn__filled s-pagination--item s-pagination--item__nav" href="#">
+                                {% icon "ArrowRight" %}
+                                <span class="v-visible-sr">page</span>
+                            </a>
```

Comment:
Related to my previous comment about not relying on `.s-btn` styles here, the styling of this button should be handled entirely with pagination styles and without any `.s-btn` styles.